src/app/campaign/inventory/inventory.component.scss

```diff
@@ -1,10 +1,5 @@
 @import '~src/sass/variables';
 
-$row-line-height: 20px;
```

Comment:
These variables all moved to the `inventory-table.component.scss`.  So weren't actually doing anything here.